api/src/test/java/bio/terra/tanagra/service/search/SqlVisitorTest.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.tanagra.service.search;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.tanagra.service.search.testing.SimpleUnderlaySqlResolver;
+import com.google.common.collect.ImmutableList;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+@Tag("unit")
+public class SqlVisitorTest {
+  private static final SearchContext SIMPLE_CONTEXT =
+      SearchContext.builder().underlaySqlResolver(new SimpleUnderlaySqlResolver()).build();
+
+  private static final Entity PERSON = Entity.builder().name("person").underlay("foo").build();
```

Comment:
What did `underlay("foo")` do here? Is this property necessary? Or is it just being ignored somehow due to the nature of the test?

Reply:
Nothing in this test. It's roughly the name for the underlying dataset, which isn't used in these filters or expressions, but would be used in a table clause, e.g. [next PR](https://github.com/DataBiosphere/tanagra/pull/5/files#diff-3a2fe285e55d6e8eb1897fcb3c79cb33634c2762f60267437759e07a2b0cce75P). I'm imagining the underlay is the "name" of the "underlying dataset", where there may be one or more backing physical BQ datasets, e.g. indexes and initial dataset. All a little speculative at this point.
added a little more to comments on Entity.